examples/Console/TestPrometheusExporter.cs

```diff
@@ -58,12 +58,11 @@ internal static async Task<object> RunAsync(int port, int pushIntervalInSecs, in
             // Application which decides to enable OpenTelemetry metrics
             // would setup a MeterProvider and make it default.
             // All meters from this factory will be configured with the common processing pipeline.
-            MeterProvider.SetDefault(Sdk.CreateMeterProvider(mb =>
-            {
-                mb.SetMetricProcessor(processor);
-                mb.SetMetricExporter(promExporter);
-                mb.SetMetricPushInterval(TimeSpan.FromSeconds(pushIntervalInSecs));
-            }));
+            MeterProvider.SetDefault(Sdk.CreateMeterProviderBuilder()
+                .SetProcessor(processor)
```

Comment:
@cijothomas do we want this to be `Set` or `Add`?

Reply:
If we allow multiple Processor, then Add. Else Set.
This decision is not clear, so I'll leave it as is until we actually implement Metric Sdk.
(This PR just make consistent pattern as Tracer only - Sdk.CreateMeterBuilder().accumulate_stuff_for_building().Build pattern)

Reply:
Roger that, thanks.